packages/kbn-repo-packages/modern/parse_package_manifest.js

```diff
@@ -118,7 +119,9 @@ function validatePackageManifestPlugin(plugin, path) {
   }
 
   const segs = path.split(Path.sep);
-  const isBuild = segs.includes('node_modules') || segs.includes('build');
+  const isBuild =
+    segs.includes('node_modules') || Path.basename(Path.dirname(repoRoot)).includes('build');
```

Comment:
I've tested that with my local setup and the issue still exists.
I've then added some debug output here and this shows me that for my path `/home/robert/github/machine-learning-qa/build/kibana_repo/examples/bfetch_explorer/kibana.jsonc` the `Path.basename(Path.dirname(repoRoot))` is `/home/robert/github/machine-learning-qa/build` (the parent directory of `repoRoot`, which still contains `build` so it fails). I think what we want to have is only looking at everything under `repoRoot` and ignore everything above it. If so, I think this is the solution:
```
const segs = Path.relative(repoRoot, path).split(Path.sep);
const isBuild = segs.includes('node_modules') || segs.includes('build');
```
In my case `Path.relative(repoRoot, path)` gives me `examples/bfetch_explorer/kibana.jsonc` which is exactly what I'd like to look at. With this it's working fine for me locally.